tests/aws/services/lambda_/event_source_mapping/test_lambda_integration_dynamodbstreams.py

```diff
@@ -782,9 +756,7 @@ def assert_events_called_multiple():
         snapshot.match("lambda-multiple-log-events", events)
 
     @markers.aws.validated
-    @pytest.mark.skipif(
-        is_v2_esm(), reason="Invalid filter detection not yet implemented in ESM v2"
-    )
+    @pytest.mark.skip(reason="Invalid filter detection not yet implemented in ESM v2")
```

Comment:
There are quite a few of these comments/references still to ESM v2 peppered around the test suites. Not necessary to remove now since it's really just aesthetic but we should consider removing these in a follow-up.

Perhaps if we decide to unpack those `*_event_source_mapping_v2` functions -> `*_event_source_mapping` in the provider it could be nice to clean up these old refs.

Reply:
that's indeed a good regression to fix timely 👍